client/src/components/new-request/analyze-pending.vue

```diff
@@ -11,8 +11,11 @@
         <NameInput is-search-again="true"/>
       </v-row>
       <v-row justify="center" class="mt-n7 mb-n3 py-0">
+        <v-col cols="12" class="detailed-search-title">
+          <b>Detailed Search</b>
```

Comment:
Invision has this as "Details Analysis".

Reply:
okay will update